account/src/main/java/io/trivial/models/binding/BaseBindingModel.java

```diff
@@ -0,0 +1,16 @@
+package io.trivial.models.binding;
+
+public class BaseBindingModel {
```

Comment:
Binding models are for user entry, so one is not needed for "id".

Please remove this class.